entrypoint.sh

```diff
@@ -46,5 +46,15 @@ install_driftctl || log_error "Fail to install driftctl"
 qflag=""
 quiet_flag
 
+# Store scan output in variable 
+scan_output="$(driftctl scan $qflag $INPUT_ARGS)"
```

Comment:
It bothers me here to have 2 scans in one action. Imagine someone tries to run but have > 10k resources in multiples states.